client/js/lounge.js

```diff
@@ -1142,13 +1207,34 @@ $(function() {
 			var chan = $(this);
 			if (chan.find(".messages .msg:not(.unread-marker)").slice(0, -100).remove().length) {
 				chan.find(".show-more").addClass("show");
+
```

Comment:
@xPaw yea, the loop I made there felt a bit 'hacky', but it was the best solution I could come up with. Had to do this because they would not get removed with those messages. If you have suggestions about what I coded here, feel free to give.